sympy/polys/ring_series.py

```diff
@@ -753,7 +753,7 @@ def rs_diff(p, x):
     for expv in p:
         if expv[n]:
             e = monomial_ldiv(expv, mn)
-            p1[e] = p[expv]*expv[n]
+            p1[e] = R.domain_new(p[expv]*expv[n])
```

Comment:
Here `p[expv]` does yield a `PythonRational`, but `expv` is a tuple and it's contents aren't of `PythonRational` type. So, one could change :
` p1[e] = R.domain_new(p[expv]*expv[n])`  --> ` p1[e] = p[expv]*R.domain_new(expv[n])` for `rs_integrate` 
and `p1[e] = R.domain_new(p[expv]*expv[n])` --> `p1[e] = p[expv]*R.domain_new(expv[n])` for `rs_diff`
These would work as well. Should I do that ?